### PR TITLE
Pstew

### PR DESCRIPTION
**Pull request by pstew**
**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Any more details?**

I was promoted to assistant member so I needed to change my department. Additionally, I am required to acknowledge our comprehensive cancer center funding under acknowledgements. Thanks!

### CODE REVIEW

File: content/14.acknowledgements.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ All other authors have declared no competing interests.
...
+We would also like to thank Fábio Madeira ([0000-0001-8728-9449](https://orcid.org/0000-0001-8728-9449)), Victor Greiff ([0000-0003-2622-5032](https://orcid.org/0000-0003-2622-5032)), Shyam Saladi ([0000-0001-9701-3059](https://orcid.org/0000-0001-9701-3059)), Anshul Kundaje ([0000-0003-3084-2287](https://orcid.org/0000-0003-3084-2287)), Brett K. Beaulieu-Jones ([0000-0002-6700-1468](https://orcid.org/0000-0002-6700-1468)), Paul Brodersen ([0000-0001-5216-7863](https://orcid.org/0000-0001-5216-7863)), Michael M. Hoffman ([0000-0002-4517-1562](https://orcid.org/0000-0002-4517-1562)), and Isaac Lazzeri for their contributions to the discussions that comprised the initial stage of the drafting process. This work has been supported in part by the Biostatistics and Bioinformatics Shared Resource at the H. Lee Moffitt Cancer Center & Research Institute, an NCI designated Comprehensive Cancer Center (P30-CA076292).
```
Comment (agitter) [inline content/14.acknowledgements.md:14]:
```suggestion
We would also like to thank Fábio Madeira ([0000-0001-8728-9449](https://orcid.org/0000-0001-8728-9449)), Victor Greiff ([0000-0003-2622-5032](https://orcid.org/0000-0003-2622-5032)), Shyam Saladi ([0000-0001-9701-3059](https://orcid.org/0000-0001-9701-3059)), Anshul Kundaje ([0000-0003-3084-2287](https://orcid.org/0000-0003-3084-2287)), Brett K. Beaulieu-Jones ([0000-0002-6700-1468](https://orcid.org/0000-0002-6700-1468)), Paul Brodersen ([0000-0001-5216-7863](https://orcid.org/0000-0001-5216-7863)), Michael M. Hoffman ([0000-0002-4517-1562](https://orcid.org/0000-0002-4517-1562)), and Isaac Lazzeri for their contributions to the discussions that comprised the initial stage of the drafting process.
```
The intended journal has a separate funding statement section, which we'll compile based on the funding information you provided in `content/metadata.yaml`.  We don't need to list this in the acknowledgements.

**Review by agitter — CHANGES_REQUESTED** [review]
Congrats on the promotion!

File: content/14.acknowledgements.md (hunk 1 above)
Comment (pstew) [reply content/14.acknowledgements.md:14]: It's a weird system, but my organization likes to track the shared resource usage through acknowledgements. We are encouraged to include the specific cores whenever possible. If it is not a big deal, I would like to please leave it in. If not, somewhere down the line I will get an email from an administrator with my chair CC'd asking me why I did not include it.

Comment (Benjamin-Lee) [reply content/14.acknowledgements.md:14]: Let's save @pstew the hassle of getting unpleasant emails and just leave it in.

**Review by Benjamin-Lee — APPROVED** [review]

File: content/14.acknowledgements.md (hunk 1 above)
Comment (agitter) [reply content/14.acknowledgements.md:14]: That's fine with me

**Review by agitter — APPROVED** [review]

**Pull request merged**